addons/explore-editor-theme/ui/IconSaver.gd

```diff
@@ -0,0 +1,34 @@
+tool
+extends VBoxContainer
+
+
+signal filesystem_changed()
+
+export var icon_name := ''
+export var type_name := ''
+
+onready var _save_dialog: FileDialog = $SaveDialog
+onready var _message: Label = $Message
+
+
+func clear_message() -> void:
```

Comment:
There is probably no need to explicitly clear this message from the parent script. You can reset it on visibility changes therefore making this node self-contained in its maintanance.

Reply:
oh that's a nice idea, thanks :+1: